R/utils.R

```diff
@@ -36,6 +38,8 @@ is_workdir_clean <- function(repo) {
 }
 
 #' Check if a vector contains valid email
+#'
+#' It only checks the format of the text, not if the email works.
```

Comment:
I've never seen an email working.  ;-)
```suggestion
#' It only checks the format of the text, not if the email address exists.
```